hack/bin/install-cabal-all.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+# copies all executables from ./dist-newstyle to ./dist
+set -euo pipefail
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+TOP_LEVEL="$(cd "$DIR/../.." && pwd)"
+DIST="$TOP_LEVEL/dist"
+
+find_exes() {
+    find ./dist-newstyle/build -executable -type f \( -path "*\/x\/*" -or -path '*\/b\/*' -or -path '*\/t\/*' \)
```

Comment:
Instead of this, we should use [`cabal-plan list-bins`](https://github.com/haskell-hvr/cabal-plan). (if we cannot use `cabal install --install-dir=<DIR>`

Reply:
Done